includes/acl/algorithm/uniformly_sampled/decoder.h

```diff
@@ -61,13 +61,14 @@ namespace acl
 
 		namespace impl
 		{
-			// TODO: Ensure RO data is aligned to 64 bytes and RW data is aligned to 64 bytes as well
-			struct DecompressionContext
+			static constexpr size_t CONTEXT_ALIGN_AS = 64;
+
+			struct alignas(CONTEXT_ALIGN_AS) DecompressionContext
 			{
 				// Read-only data
-				const uint32_t* default_tracks_bitset;
```

Comment:
In fact, if we mark the context as final and call the destructor on the fully qualified type, the vtable will not be touched at all.